core/new-gui/src/app/workspace/types/execute-workflow.interface.ts

```diff
@@ -89,7 +89,8 @@ export enum OperatorState {
 export interface OperatorStatistics extends Readonly<{
   operatorState: OperatorState,
   aggregatedInputRowCount: number,
-  aggregatedOutputRowCount: number
+  aggregatedOutputRowCount: number,
+  aggregatedOutputResults: ResultObject | undefined,

```

Comment:
This would be the same comment, can we decouple `aggregatedOutputResults` from `OperatorStatistics`, or possibly rename `OperatorStatistics`?

Reply:
We'll keep the name for now...